packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -131,6 +131,29 @@ export interface AnchorEvents {
 	 */
 	subtreeChanging(anchor: AnchorNode): PathVisitor | void;
 
+	/**
+	 * Emitted when exiting the subtree rooted at `anchor` during a delta visit, which means that all effects of that
+	 * visit in that subtree are now visible. HOWEVER, as of 2024-03-27, applying changes to a tree requires two visits
+	 * (the detach and the attach passes) so this event fires twice.
+	 *
+	 * @remarks
+	 * Note that this event naturally bubbles up the tree but in a pattern resembling breadth-first traversal, not
+	 * depth-first traversal, like other events might do.
+	 * For example, if node A is the parent of nodes B and C and a batch of changes affects both B and C, this event will
+	 * fire on B and C first in some order, and only then will it fire on A.
+	 * Additionally, it will fire on A *only once*, not once after it fires on B and again after it fires on C.
```

Comment:
This is based on implementation details and should not be part of the doc/contract.